assignments/sa/websockets/index.md

```diff
@@ -206,7 +211,7 @@ io.on('connection', (socket) => {
 ```
 
 
-Ok we have some basic functionality now. Let's work on the frontend for a little bit. We'll come back and add more listeners here. Don't forget to start up mongod and also `npm run dev` the server.
+Ok we have some basic functionality now. Let's work on the frontend for a little bit. We'll come back and add more listeners here. Don't forget to start up mongod and also `npm start` the server.
```

Comment:
'run dev' didn't work for me so i changed it to start! (but maybe mine was set up incorrectly?)

Reply:
this was confusing for people so i made it npm start just like the frontend, good find!